src/XmlAgent.cc

```diff
@@ -30,7 +30,8 @@ XmlAgent::XmlAgent() : SCRAgent()
  */
 XmlAgent::~XmlAgent()
 {
-    xmlCleanupParser ();
+    // bsc#1047449, this crashes if ruby uses nokogiri (which also uses libxml2)
```

Comment:
please add also link or quote libxml documentation which mention why it is not problem to omit it and in fact recommended solution